build/stf-run-ci/.ansible-lint

```diff
@@ -0,0 +1,52 @@
+---
+profile: null
+skip_list:
```

Comment:
Lol. Are we currently in such a bad shape that we fail all of these?!

Reply:
I just wanted to keep the scope incredibly small and focused on `fqcn`. There were about 57 total issues if I remember correctly. Nothing too crazy, but I got this list by doing `ansible-lint -L | cut -f3 -d' '` and then adding them to the list, and removing `fqcn`. It then complained about 2 more non-optional items, and those ones also pass :)

After we get through all of this I'd like to remove the list and start fixing anything it finds. It's not _that_ bad, but didn't want to over scope :)

Reply:
We don't fail all of these, these are all the rules that are checked